appveyor.yml

```diff
@@ -0,0 +1,32 @@
+# version format
+version: 1.0.{build}-{branch}
+
+# Do not build on tags (GitHub and BitBucket)
+skip_tags: true
+
+# Do not build feature branch with open Pull Requests
+skip_branch_with_pr: true
```

Comment:
I think Travis actually will run on external PRs it just won't pass any secrets.